lxd/storage/drivers/utils_cgo.go

```diff
@@ -265,6 +265,7 @@ import "C"
 // LoFlagsAutoclear determines whether the loop device will autodestruct on last
 // close.
 const LoFlagsAutoclear int = C.LO_FLAGS_AUTOCLEAR
+const LoFlagsDirectIO int = C.LO_FLAGS_DIRECT_IO
```

Comment:
The commit message for this one is incorrect as it only really adds the const.

Reply:
I'll get that changed